jwcrypto/jwe.py

```diff
@@ -390,9 +396,10 @@ def __init__(self, hashsize, keysize):
         self.keysize = keysize // 8
 
     def _get_key(self, alg, key, p2s, p2c):
-        if key.key_type != 'oct':
-            raise InvalidJWEKeyType('oct', key.key_type)
-        plain = base64url_decode(key.get_op_key('encrypt'))
+        if isinstance(key, bytes):
```

Comment:
I thk in this case I really prefer the more verbose form, it's hard to understand what the compact form does at a glance